### PR TITLE
Charzy portfolio/project gallery

### PR DESCRIPTION
**Pull request by Charzy-1**
html and css files was created and codes were added respectively
linters were configured but abolutely unable to fix the bug that pops up on stylint
readme was modified but not yet complete
The interface i try to build is not yet aligned.

### CODE REVIEW

**Comment by Charzy-1** [comment]
Hi @basitali111 and mkp please rescue me in fixing the stylint linters and a betterway to code in getting the template  i try to get. Also making my site responsive is tough.

Thanks for your responses in advance

**Comment by basitali111** [comment]
Kindly share me timings when you are available ?

**Comment by Charzy-1** [comment]
Ok. We can meet the same time (8pm) we normally use to meet.

Thanks & Regards

**Comment by basitali111** [comment]
Hello,

I understand. I will see you then. However, I would like to encourage you
to try to resolve the issue on your own. You have done most of the research
and error-solving for your project, so I believe you are capable of
handling this.

Please try your best to resolve the issue before our meeting. This will
help us avoid wasting time and repetition.

Thank you for your understanding.

Best regards
Basit Ali

**Comment by Charzy-1** [comment]
I will always try my best. The error message is almost same for the various
style block. I know that when I am able to fix one, I will be 70% good to
go in fixing others. I just still hovering around it.

See you soon.

**Comment by Charzy-1** [comment]
Hi. @basitali111 Thanks for your assistance thus far.

Please have a recheck on the pull request again and give me feedback.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
```
Comment (basitali111) [inline styles.css]: - *{
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}
- Kindly try to use this code on top of your stylesheet to git rid of the extra by default space around your webpage so your design become better

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
Apex_1715419212206
- **Its your design** 
- **But the design you have to make is this:**
Apex_1715419473638
- **remember, it should be responsive here is project requirements on it see it carefully and follow the instruction**
-The website should be responsive, following these breakpoints:
- Minimum mobile screen size should be 375px
- Minimum desktop screen size should be 768px
- Use responsive values in your CSS rules, like percentages (i.e. width: 80%),
instead of pixels



There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Thanks for the feedback! I will correct it

**Comment by Charzy-1** [comment]
Hi @basitali111  
I have made some necessary corrections on the first part of the assignment. Kindly have a look.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+a:hover {
+  background: gray;
+}
+
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+  padding: 20px;
+}
```
Comment (basitali111) [inline styles.css]: - Kindly already we told do not use width in px always use width in percentage so your webpage become responsive and also remove padding and margin to remove extra white space around the webpage

File: images/lnkdn.jpg
Comment (basitali111) [inline images/lnkdn.jpg:1]: - one more thing kindly try to download this file in correct way so you can use as same as in figma

[Apex_1715628168749.webm](https://github.com/Charzy-1/My-Portfolio-Website/assets/63063973/2e6f0678-215c-4718-befb-eb1241299657)

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
Apex_1715419212206
Apex_1715627208671

#  **Its your design for mobile and desktop  ** 
# But the design you have to make is this:
## For Desktop:
Apex_1715419473638
## For Mobile
- Make a hamburger menu so in mobile size when it click menu appear and when you click it goes
- For Helping try to use this to research on youtube how to make this ( **How to create a responsive navbar with hamburger menu** )
Apex_1715627109076

- **remember, it should be responsive here is project requirements on it see it carefully and follow the instruction**
-The website should be responsive, following these breakpoints:
- Minimum mobile screen size should be 375px
- Minimum desktop screen size should be 768px
- Use responsive values in your CSS rules, like percentages (i.e. width: 80%),
instead of pixels



There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Thanks for the feedbak. I will appreciate if you can review the second pull request so i can see to the corrections tommorrow. Meanwhile, can we meet so i will be assisted in figuring out some things going forward?

**Comment by Charzy-1** [comment]
Hi @basitali111. Thanks thus far
Kindly have a look of my assignment 1 again.

**Comment by Charzy-1** [comment]
Please don't review yet let me finish the mobile version

**Comment by basitali111** [comment]
Ok, no problem.

**Comment by Charzy-1** [comment]
Please is it possible for us to meet? I want you to assist me in fixing
some error i ran into trying to make a mobile version

**Comment by Charzy-1** [comment]
Hi @basitali111 i have tried the best i could to do get what was required. Only the placement of the banner image on my mobile is what i couldn't figure out. Please review my code and see if it is what i can go ahead with.

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @Charzy-1 ,

- First of all, in your navbar in mobile size the `menu `should be hidden and only appear when the user clicks on 3 dots on the navbar in mobile size  so in my recommendation what you can do is use `display: none` in `media query` and hide the menus of desktop and make a separate menu for mobile
Apex_1715794257519

-  Secondly, in the navbar, your menu is not equally centered kindly try to center it.
Apex_1715793223200
- thirdly your website is not taking up whole space on the desktop size I think you can work on height also and make height responsive
Apex_1715793481717
- Also fix your image by doing some research because it's good for you in the real world these minors even junior programmers have to solve by themselves if you still find it difficult let me know we arrange a meeting.
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff
# Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ Linters pass
## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Charzy-1** [comment]
Hi @basitali111 . kindly review my work.

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
## You are almost finished. Please work on the mobile version.
Apex_1716013607462
## Here is your design. You just have to work on the icons and mobile image to make them responsive.
Apex_1716014080259

There are still some issues that need to be addressed before moving on to the next project, but you are almost there!

giphy_2


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Hi @basitali111 
Please check again. Thanks

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
## You are almost finished. Please You need to still work on the mobile version Specially on icons because when I go to mobile size the icons goes far from text kindly fix that.


## Also work on the humburgue menu because it is not look like Figma design kindly  look at that and try to make like that:







There are still some issues that need to be addressed before moving on to the next project, but you are almost there!

giphy_2


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Ok. Please check again. @basitali111

**Comment by Charzy-1** [comment]
Hi. Please check my code again. Thanks

**Comment by Charzy-1** [comment]
Hi. I am still here

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
## You are nearly done. Please work on the mobile version, especially the icons. When I switch to mobile size, the icons move far from the text. Please fix this.
### Recommendation:
- During checking, ensure that the content fits all mobile sizes, including small mobiles where the main issues are with icons and images.
Apex_1716141293302

## Design that you have to make is this :
Apex_1716140815189


## Work on the hamburger menu as it doesn't match Figma's design. Please review and adjust accordingly.

Apex_1716140694538






There are still some issues that need to be addressed before moving on to the next project, but you are almost there!

giphy_2


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Please kindly check for the modification of image, social icon respnosiveness and hambuger @basitali111

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️
Hi @Charzy-1 ,
Good job so far!
## You are almost finished. Please work on the mobile version, particularly the icons. When I switch to mobile size, the icons move too far from the text. Please address this. Also, please review the following video.

[Apex_1716186312659.webm](https://github.com/Charzy-1/My-Portfolio-Website/assets/63063973/6f98a16f-2f3a-4a52-84c9-5f1ac912f469)


There are still some issues that need to be addressed before moving on to the next project, but you are almost there!

giphy_2


## Highlights
✔️ No Linters errors
✔️ Professional Readme
✔️ Correct Gitflow

## Required
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Charzy-1** [comment]
Hi @basitali111 I have modified the CSS with the recommendations given for the mobile responsiveness

**Review by basitali111 — APPROVED** [review]
# Approve 🟢 🟢 🟢 
Hi @Charzy-1,  

Your project is complete!  There is nothing else to say other than... it's time to merge it :ship: 

- [Optional] Kindly ensure that your pull request title and description are well-written next time."
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @basitali111.

**Comment by Charzy-1** [comment]
Thanks for the feedback!

**Pull request merged**